### ISSUE TITLE
Custom Overlay values & spells with different timers

### ISSUE THREAD

**Issue by mspykerez** [rgd87/NugRunning#38]
Entering custom values in 'Overlay Start' and 'Overlay End' does not work properly for spells that can have different timers such as rogue's 'Slice and Dice' and 'Rupture' which have a timer based on the number of combo points spent.
Also for such spells the 'Calculate pandemic overlay from duration' is not 100% accurate because like I said before the timer can vary. It would be nice thou if the pandemic could be automatically calculated dynamically based on the number of combo points spent, but I'm not sure if that's possible to do.

**Comment by d87** [comment]
Yeah, It doesn't work because it auto recalculates overlay to max cp duration based on whether you have Deeper Stratagem or not.
The issue is mostly that the only way to know how much CP you spent on a finisher is pretty sketchy and especially if it fails to take into account things like 7cp rupture from 2cp echoing reprimand it may end up even worse than just static max cp number which is at least always close to what you will realistically apply
